cumulusci/cli/tests/test_cci.py

```diff
@@ -329,6 +329,15 @@ def test_service_connect(self):
 
         run_click_command(cmd, project=False)
 
+    def test_service_connect_invalid_org(self):
+        multi_cmd = cci.ConnectServiceCommand()
+        ctx = mock.Mock()
+        config = mock.Mock()
+        config.project_config.services__test__attributes = {"foo": {}}
+
+        with self.assertRaises(click.UsageError):
+            multi_cmd.get_command(ctx, config)
```

Comment:
`get_command` takes two params: ctx (the context object used by the `click` library -- in this case we're passing a mock object because it's not actually used and that's easier than setting up a real context) and a string name of the command to get. So you want to pass a string here rather than `config` ... either "test" if you want it to succeed or something else if you want it to fail.

So how do you tell the system to use the mock config object you just created? That's what the `with mock.patch("cumulusci.cli.cci.TEST_CONFIG", config):` block is for in other tests. It temporarily sets the TEST_CONFIG variable in the cci module to our config object, and if that variable is set then the load_config function will use it instead of looking for config files on the hard disk (which is where your test is currently breaking).